adapters/adapters-commons/src/test/java/uk/gov/justice/raml/common/mapper/ActionMappingTest.java

```diff
@@ -73,6 +73,38 @@ public void shouldCreateMappingsCollections() throws Exception {
                 allOf(hasProperty("requestType", equalTo("application/vnd.bbbbb+json")), hasProperty("name", equalTo("actionB")))));
     }
 
+    @Test
+    public void shouldCreateMappingIfPrefixedByOtherText() throws Exception {
+
+        List<ActionMapping> mappings = listOf("test" + mappingDescriptionWith(
```

Comment:
Rather than "test" could we use something more descriptive like "Description of action" / Action A